File: c++/Easy/Part001/Uppercase Letter Count.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string S;
    
    // Input for string S
    getline(cin, S);
    
    int upperCaseCount = 0;
    
    // Loop through each character in the string
    for (char c : S) {
        if (isupper(c)) {
            upperCaseCount++;
        }
    }
    
    // Print the count of uppercase letters
    cout << upperCaseCount << endl;

    return 0;
}

File: python/Easy/Part002/Uppercase Letter Count.py
def main():
    # Input for string S
    S = input().strip()
    
    upper_case_count = 0
    
    # Loop through each character in the string
    for c in S:
        if c.isupper():
            upper_case_count += 1
    
    # Print the count of uppercase letters
    print(upper_case_count)

if __name__ == "__main__":
    main()
